css/styles-drag-drop.css

```
@@ -0,0 +1,23 @@
+
+.spacing{
+    height: 20px;
+  }
```

```suggestion
hr {
  margin: 50px 0 50px 0px;
}
```

Puts more space between the calculator and the drag and drop 😉 .